### DISCUSSION TITLE
Falling back to the default storyStort

### DISCUSSION THREAD

**Discussion by marvhen** [storybookjs/storybook#14526] [Help]
When [Sorting Stories](https://storybook.js.org/docs/angular/writing-stories/naming-components-and-hierarchy#sorting-stories) with a custom storySort function, is there anyway to fallback to the default sort function?  

(i.e. the one that would have been called if a custom sortySort had not been specified)

e.g.
```
// .storybook/preview.js

export const parameters = {
  options: {
    storySort: (a, b) => {
      if (something)
        return 0;
      else
        return storySortDefault(a, b);
    }
  },
};
```

It looks like it is defined in [storySort.ts](https://github.com/storybookjs/storybook/blob/next/lib/client-api/src/storySort.ts) but I don't see a way to get at it.

**Comment by michaelaflores** [comment]
Does `something` depend on the arguments provided to `storySort`? If not, I would do:

```js
const parameters = {
  options: {},
};

if (!something) {
   parameters.options.storySort = (a, b) => storySortDefault(a, b);
}

export { parameters }
```

**Reply by marvhen** [reply]
Hmmm....I don't follow you there...

You reference "storySortDefault", but that is just a name I used as an example of the function I am looking for.
